source/ns_cmdline.c

```diff
@@ -81,14 +81,39 @@
 
 #define TRACE_GROUP "cmdL"
 
+#ifndef MBED_CMDLINE_BOOT_MESSAGE
+#define MBED_CMDLINE_BOOT_MESSAGE "ARM Ltd\r\n"
+#endif
+#define ESCAPE(x) "\x1b" x
+#define CR_S "\r"
+#define LF_S "\n"
+#define CLEAR_ENTIRE_LINE ESCAPE("[2K")
+#define CLEAR_ENTIRE_SCREEN ESCAPE("[2J")
+#define ENABLE_AUTO_WRAP_MODE ESCAPE("[7h")
+#define MOVE_CURSOR_LEFT_N_CHAR ESCAPE("[%dD")
+
+#define SET_TOP_AND_BOTTOM_LINES ESCAPE("[;r")
+#define MOVE_CURSOR_TO_BOTTOM_RIGHT ESCAPE("[999;999H")
+#define STORE_CURSOR_POS ESCAPE("7")
+#define RESTORE_CURSOR_POS ESCAPE("8")
+#define GET_CURSOR_POSITION ESCAPE("[6n")
+
+#define REQUEST_SCREEN_SIZE STORE_CURSOR_POS SET_TOP_AND_BOTTOM_LINES MOVE_CURSOR_TO_BOTTOM_RIGHT GET_CURSOR_POSITION RESTORE_CURSOR_POS
+
 /*ASCII defines*/
 #define ESC 0x1B
 #define DEL 0x7F
 #define BS  0x08
 #define ETX 0x03
+#define ETB 0x17
 #define TAB 0x09
 #define CAN 0x18
 
+#define DEFAULT_RETFMT "retcode: %i\r\n"
+#define DEFAULT_PROMPT "/>"
```

Comment:
`DEFAULT_PROMPT` is not shown, only when serial get something input after `reset` button pressed.
it's just looking like below, only when typing something, "/>" will appear
```
ARM Ltd
[INFO][main]: write 'help' and press ENTER
```

Reply:
I saw it also when playing with linux test app - have to lookup it more deeploy..

Reply:
seems to work just fine with mbed device - Related only linux example I think.